src/lib.rs

```diff
@@ -497,6 +478,15 @@ impl<K: EnrKey> Enr<K> {
             return Err(EnrError::UnsupportedIdentityScheme);
         }
 
+        // TODO @age: You can't insert low integers with Enr::insert API now
+        // A solution would be to change the fn signature of ::insert to
+        // `pub fn insert<T: Encodable>(.., value: T)` so that u16 are encoded properly
+
+        // Prevent inserting invalid RLP integers
```

Comment:
in the future we should also check for the other reserved values that are not integers